generate.js

```diff
@@ -134,6 +141,22 @@ function generate(args) {
                 mkdirParentsSync(apiOutputDir);
             targetMaker.makeCombinedAPI(allApis, targetSourceDir, apiOutputDir);
         }
+
+        if (targetMaker.makeGameServerAPI) {
+            apiOutputDir = targetMaker.putInRoot ? sdkOutputDir : path.resolve(sdkOutputDir, "PlayFabSDK");
+            console.log(" + Generating GameServer to " + apiOutputDir);
+            if (!fs.existsSync(apiOutputDir))
+                mkdirParentsSync(apiOutputDir);
+            targetMaker.makeGameServerAPI(gameServerApis, targetSourceDir, apiOutputDir);
+        }
+
+        if (targetMaker.makeStrangeIoC) {
```

Comment:
My first reaction to makeStrangeIoC in generate.js is negative. This seems sdk-specific, which is the entire point of the make.js files.